### PR TITLE
[NetAppFiles] Add 2020-11-01 swagger changes

### PR DESCRIPTION
**Pull request by audunn**
This updates the NetApp Files SDK to swagger api-version 2020-11-01 
https://github.com/Azure/azure-rest-api-specs/pull/12388
Including a fix here https://github.com/Azure/azure-rest-api-specs/pull/12852

# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [x] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [x] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version,

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

**Comment by audunn** [comment]
Note we had issues with generated validation code https://github.com/Azure/autorest/issues/3585   that was fixed in code this now triggered a build validation so was not longer feasible. 

However that seems to have been fixed and we can use that fix by running autorest with `--use:@microsoft.azure/autorest.csharp@2.3.90` like so:

`autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/netapp/resource-manager/readme.md --csharp --version=v2 --use:@microsoft.azure/autorest.csharp@2.3.90 --reflect-api-versions --csharp-sdks-folder=C:\Users\****\Source\repos\azure\azure-sdk-for-net\sdk`

Looks like this shows in the pipeline as it thinks generated code was changed, Is there was way to update the tooling to use this autorest cmd line?

**Comment by audunn** [comment]
I attempted to update generate.ps1 to correct above issue, that does not seem to work. The validation tool seem sto use its own autorest command.

**Comment by audunn** [comment]
> @allenjzhang and others
> Note we had issues with generated validation code (not checking for null values) causes issues for SDK clients such as powershell. See: [Azure/autorest#3585](https://github.com/Azure/autorest/issues/3585)
> However that issue seems to have been fixed and we can use that fix that issue by running autorest with
> `--use:@microsoft.azure/autorest.csharp@2.3.90` like so:
> 
> `autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/netapp/resource-manager/readme.md --csharp --version=v2 --use:@microsoft.azure/autorest.csharp@2.3.90 --reflect-api-versions --csharp-sdks-folder=C:\Users\****\Source\repos\azure\azure-sdk-for-net\sdk`
> 
> Now it looks like this shows in the pipeline as it thinks generated code was changed. It was not changed in this case. The tool in the pipeline probably uses generate.ps1 that creates a slightly different autorest command line command that would include this validation bug (not checking for nulls).
> Is there a way to update the tooling to use this autorest cmd line or resolve this pipeline issue some other way?
> 
> update: I tried to update the generate.ps1 script and ran locally with updated autorest command that worked. But it seems like the validation tool is using some other autorest command so that does not fix the issue.

@allenjzhang can you assist with this issue?

File: sdk/netapp/Microsoft.Azure.Management.NetApp/src/Microsoft.Azure.Management.NetApp.csproj [hunk 1]
```diff
@@ -13,10 +13,13 @@
     <PackageTags>MicrosoftAzure Management;NetApp</PackageTags>
```
Comment (allenjzhang) [inline sdk/netapp/Microsoft.Azure.Management.NetApp/src/Microsoft.Azure.Management.NetApp.csproj:13]: Please increment the version to 1.10.0

**Review by allenjzhang — CHANGES_REQUESTED** [review]
Please update version in csproj.

**Review by allenjzhang — APPROVED** [review]

**Comment by allenjzhang** [comment]
Code verification result was incorrect. Ignored and merged.

**Pull request merged**